src/main/java/org/reaktivity/nukleus/http_cache/internal/stream/util/GroupThrottle.java

```diff
@@ -55,21 +55,20 @@ private void updateThrottle()
             long diff = newLowWaterMark - groupWaterMark;
             if (diff > 0)
             {
-                // TODO, track frames?
-                writer.doWindow(connectReply, connectReplyStreamId, (int)diff, (int)diff);
+                writer.doWindow(connectReply, connectReplyStreamId, (int)diff, 0);
```

Comment:
`padding` intentionally `0` ?

Reply:
I think we need max padding across streams ?